### PR TITLE
Feature dispatch actions

### PR DESCRIPTION
**Pull request by Ephrame**

### CODE REVIEW

File: src/behaviours/form.js [hunk 1]
```diff
@@ -6,13 +6,30 @@ import {inputChange, inputBlur, inputBlurList} from '../actions/input';
 import find from 'lodash/find';
 import compose from 'lodash/flowRight';
 import isString from 'lodash/isString';
+import isObject from 'lodash/isObject';
```
Comment (pierr) [inline src/behaviours/form.js]: :+1:

File: src/example/components/user-form.js [hunk 2]
```diff
@@ -87,7 +101,7 @@ const ConnectedUserFormConfig = compose(
 
 function ComponentUser(props)  {
   return <div>
-    <ConnectedUserFormConfig {...props}/>
+  hjghjgjhghj
```
Comment (pierr) [inline src/example/components/user-form.js]: wtf

File: src/example/components/user-form.js [hunk 3]
```diff
@@ -12,26 +12,38 @@ import LineComponent from '../../components/line'
 import Panel from '../../components/panel';
 import compose from 'lodash/flowRight';
 
+
+export const deleteFields = (victoire) => {
+  type: MY_ACTION,
+  victoire
```
Comment (pierr) [inline src/example/components/user-form.js]: victoire?

File: src/behaviours/form.js (hunk 1 above)
Comment (Ephrame) [inline src/behaviours/form.js]: :+1:

Comment (Ephrame) [inline src/behaviours/form.js]: :+1:

**Pull request merged**